<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fpChart_II</class>
 <widget class="QMainWindow" name="fpChart_II">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>933</width>
    <height>708</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Кислородомер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="minimumSize">
         <size>
          <width>298</width>
          <height>121</height>
         </size>
        </property>
        <property name="title">
         <string>Значение</string>
        </property>
        <widget class="QLabel" name="lblVal_1">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>23</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_percent">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblVal_4">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>70</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_mg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="numDigits">
          <number>8</number>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>298</width>
          <height>121</height>
         </size>
        </property>
        <property name="title">
         <string>Минимальное  значение</string>
        </property>
        <widget class="QLabel" name="lblVal_5">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>70</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_percent_min">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblVal_2">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>23</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_mg_min">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>298</width>
          <height>121</height>
         </size>
        </property>
        <property name="title">
         <string>Максимальное  значение</string>
        </property>
        <widget class="QLabel" name="lblVal_6">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>70</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_percent_max">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="lblVal_3">
         <property name="geometry">
          <rect>
           <x>197</x>
           <y>23</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdMain_mg_max">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>41</height>
          </size>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QwtPlot" name="qwtPlot">
        <property name="minimumSize">
         <size>
          <width>911</width>
          <height>281</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>221</height>
         </size>
        </property>
        <property name="title">
         <string>Параметры отображения</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QCheckBox" name="chbPause">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>46</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Пауза отображения</string>
         </property>
        </widget>
        <widget class="QToolButton" name="tbResetViewPar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chbZoom">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Увеличение</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>731</width>
          <height>235</height>
         </size>
        </property>
        <property name="title">
         <string>Маркеры</string>
        </property>
        <widget class="QToolButton" name="tbAddMarker">
         <property name="geometry">
          <rect>
           <x>124</x>
           <y>197</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>98</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="leVal_marker">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>98</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="leComment_marker">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>132</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>132</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string>Пояснение</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbCon">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>37</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbTime">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>75</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Время</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>23</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>-5</number>
         </property>
         <property name="text">
          <string>Список маркеров</string>
         </property>
        </widget>
        <widget class="QToolButton" name="tbDelMarker">
         <property name="geometry">
          <rect>
           <x>465</x>
           <y>197</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
        <widget class="QListWidget" name="lwListMarker">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>50</y>
           <width>391</width>
           <height>141</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>391</width>
           <height>141</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="muAbout">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <addaction name="muAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actConf">
   <property name="text">
    <string>Градуировка</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
